src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Common/BuildOutput.cs

```diff
@@ -50,7 +50,7 @@ public bool IsTargetSkipped (string target)
 			bool found = false;
 			foreach (var line in Builder.LastBuildOutput) {
 					found = line.Contains (string.Format ("Target {0} skipped due to ", target))
-					            || line.Contains (string.Format ("Skipping target \"{0}\" because it has no outputs.", target))
+					            || line.Contains (string.Format ("Skipping target \"{0}\" because it has no ", target)) //NOTE: message can say `inputs` or `outputs`
```

Comment:
I half wonder if we should start using interpolated strings here...  But that might look inconsistent. :-/

Reply:
If we do, we should change all of them.

Reply:
I’ll change all these to use interpolated strings. It would look nicer.